exporter/opentelemetry-exporter-zipkin/tests/test_zipkin_exporter.py

```diff
@@ -289,11 +286,21 @@ def test_export(self):
             status = exporter.export(otel_spans)
             self.assertEqual(SpanExportResult.SUCCESS, status)
 
-        mock_post.assert_called_with(
-            url="http://localhost:9411/api/v2/spans",
-            data=json.dumps(expected),
-            headers={"Content-Type": "application/json"},
+        # pylint: disable=unsubscriptable-object
+        kwargs = mock_post.call_args[1]
+
+        self.assertEqual(kwargs["url"], "http://localhost:9411/api/v2/spans")
+        exported_spans = sorted(
+            json.loads(kwargs["data"]), key=lambda span: span["timestamp"]
         )
+        for expected, exported in zip(expected_spans, exported_spans):
```

Comment:
nit: compating expected and exported is hard to do visually as the words are so similar

Reply:
Renamed to expected and actual.